src/code2vec.py

```diff
@@ -18,8 +20,10 @@ def code2vec(args):
         .link(UastRow2Document()) \
```

Comment:
Add `Moder` transformer before `Row2Document`, since embedding will probably be at function level, not file level (I think there is no need to add and argument, just initialize the transformer with `func` string)